definitions/divorce/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -932,6 +932,18 @@
     "PageDisplayOrder": 1,
     "ShowSummaryChangeOption": "Yes"
   },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
I don't know why it's here. It was a part of:
https://github.com/hmcts/div-ccd-definitions/pull/423

already merged into master. Should not be displayed here.